responsiveness added

Vicentefg10 opened the pull request:
responsiveness added

Commit:
responsiveness added

mannsn commented on css/index.css:
```diff
@@ -95,3 +95,59 @@ h1#name {
+
+/*Mobile - first*/
+
+body {
```
This is just adding to the body in line 1, but the mobile first starts at line 1, so you could have just added to the body section starting at line 1

mannsn commented on css/index.css:
```diff
@@ -95,3 +95,59 @@ h1#name {
+  font-size: 1em;
+}
+
+h1 {
```
This is same as font size earlier for h1 , this is not really doing anything.

mannsn commented on css/index.css:
```diff
@@ -95,3 +95,59 @@ h1#name {
+  font-size: 2em;
+}
+
+section {
```
This is same as padding for section earlier in file, this is not really doing anything.

mannsn commented on css/index.css:
```diff
@@ -95,3 +95,59 @@ h1#name {
+/*Tablet View*/
+@media screen and (min-width: 600px) {
+  h1 {
+    font-size: 2.5em;
```
Assignment was looking for two style changes per element, so need an additional style change here.

mannsn commented on css/index.css:
```diff
@@ -95,3 +95,59 @@ h1#name {
+  }
+
+  section {
+    padding: 20px;
```
Assignment was looking for two style changes per element, so need an additional style change here.

mannsn commented on css/index.css:
```diff
@@ -95,3 +95,59 @@ h1#name {
+    padding: 20px;
+  }
+
+  nav ul {
```
Assignment was looking for two style changes per element, so need an additional style change here.

mannsn commented on css/index.css:
```diff
@@ -95,3 +95,59 @@ h1#name {
+
+/* Desktop View*/
+@media screen and (min-width: 1024px) {
+  h1 {
```
Assignment was looking for two style changes per element, so need an additional style change here.

mannsn commented on css/index.css:
```diff
@@ -95,3 +95,59 @@ h1#name {
+    font-size: 3.5em;
+  }
+
+  section {
```
Assignment was looking for two style changes per element, so need an additional style change here.

mannsn commented on css/index.css:
```diff
@@ -95,3 +95,59 @@ h1#name {
+    padding: 30px;
+  }
+
+  nav ul {
```
Assignment was looking for two style changes per element, so need an additional style change here.

mannsn reviewed (commented):
The assignment asked for the following: Make a minimum of two changes to at least three elements for each of your media queries.  Example: In your tablet media query, change the font size (1st change) and color (2nd change) of your headers (1st element), change the padding or margin (1st change) and background color (2nd change) of your unordered lists (2nd element), and change the font size (1st change) and font style (2nd change) of your name at the top of your page (3rd element)



From AI reviewer:

Determine if you'd like to build your site mobile-first or desktop-first

While the mobile-first approach is indicated through the order of your media queries, make sure to clearly state within a comment at the start of your CSS file that you are adopting a mobile-first approach. This clarification helps in understanding your design intentions.

Make a minimum of two style properties changes to at least three html elements for each of your media queries

Currently, you've made changes to only the h1, section, and nav ul elements in your media queries. Ensure that each media query modifies at least three different HTML elements and involves at least two style properties for each element. For instance, consider adding or adjusting styles for nav, #Experience ul, and other sections to fully meet the requirements.

Summary of Good Work
You've effectively used media queries to adjust the layout for different viewports, capturing the transition from mobile to desktop views.
Specific efforts towards organizing CSS with comments for different screen sizes are evident and appreciated.
Integrating grid layout into sections showcases an attempt towards stretch goals, which is a constructive and notable approach.
The HTML is well-structured with semantic tags and includes appropriate metadata in the head section. The use of lists and sections makes the content clear and organized.

Vicentefg10 commented:
Hi,

I made the changes based on your feedback. I added the second style change where needed, like centering the headings in the tablet view and adjusting padding and background colors. For the desktop view, I increased the font size, added text-transform, and updated the background color for the navigation.

I also threw in a third change since I thought it fit the design well. I’m still figuring out how to center the experience section with grid, but I’m getting there!

Thanks! I’ll be waiting for any more feedback.
